sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/samples/Sample2_GetCertificates.cs

```diff
@@ -12,9 +12,9 @@
 namespace Azure.Security.KeyVault.Certificates.Samples
 {
     /// <summary>
-    /// Sample demonstrates how to list keys and versions of a given key,
-    /// and list deleted keys in a soft-delete enabled Key Vault
-    /// using the synchronous methods of the KeyClient.
+    /// Sample demonstrates how to list certificates and versions of a given certificates,
```

Comment:
This is repeated in the readme. Consider ways to not repeat the text.

Reply:
These files may be viewed at different times. Each is written to standalone.